src/shorthands.jl

```diff
@@ -104,3 +104,215 @@ zlims!(lims::NTuple{2, Real}) = zlims!(current_scene(), lims)
 xlims!(lims::Real...) = xlims!(current_scene(), lims)
 ylims!(lims::Real...) = ylims!(current_scene(), lims)
 zlims!(lims::Real...) = zlims!(current_scene(), lims)
+################################################################################
+"""
+    ticklabels(scene)
+
+Returns the all the axis tick labels.
+"""
+function ticklabels(scene)
+    axis = scene[Axis]
+    @assert !isnothing(axis)
+    return axis.ticks.ranges_labels[][2]
+end
+
+"""
+    xticklabels(scene)
+
+Returns the all the x-axis tick labels. See also [`ticklabels`](@ref).
+"""
+xticklabels(scene) = labels(scene)[1]
+
+"""
+    yticklabels(scene)
+
+Returns the all the y-axis tick labels. See also [`ticklabels`](@ref).
+"""
+yticklabels(scene) = labels(scene)[2]
+
+"""
+    zticklabels(scene)
+
+Returns the all the z-axis tick labels. See also [`ticklabels`](@ref).
+"""
+zticklabels(scene) = labels(scene)[3]
+
+"""
+    tickranges(scene)
+
+Returns the tick ranges along all axes.
+"""
+function tickranges(scene)
+    axis = scene[Axis]
+    @assert !isnothing(axis)
+    scene[Axis].ticks.ranges_labels[][1]
+end
+
+"""
+    xtickrange(scene)
+
+Returns the tick range along the x-axis. See also [`tickranges`](@ref).
+"""
+xtickrange(scene) = tickranges(scene)[1]
+
+"""
+    ytickrange(scene)
+
+Returns the tick range along the y-axis. See also [`tickranges`](@ref).
+"""
+ytickrange(scene) = tickranges(scene)[2]
+
+"""
+    ztickrange(scene)
+
+Returns the tick range along the z-axis. See also [`tickranges`](@ref).
+"""
+ztickrange(scene) = tickranges(scene)[3]
+
+"""
+    ticks!([scene,]; tickranges=tickranges(scene), ticklabels=ticklabels(scene))
+
+Set the tick labels and ranges along all axes. The respective labels and ranges
+along each axis must be of the same length.
+"""
+function ticks!(scene; tickranges=tickranges(scene), ticklabels=ticklabels(scene))
+    axis = scene[Axis]
+    @assert !isnothing(axis)
+    # Have to set `ranges_labels` first so that any changes in the length of these
+    # is reflected there first.
+    axis.ticks.ranges_labels[] = (tickranges, ticklabels)
+    axis.ticks.ranges[] = tickranges
+    axis.ticks.labels[] = ticklabels
+    return nothing
+end
+function ticks!(; tickranges=tickranges(scene), ticklabels=ticklabels(scene))
+    ticks!(current_scene(), ranges=ranges, labels=labels)
+end
+
+"""
+    xticks!([scene,]; xtickranges=xtickrange(scene), xticklabels=xticklabel(scene))
+
+Set the tick labels and range along the x-axes. See also [`ticks!`](@ref).
+"""
+function xticks!(scene; xtickrange=xtickrange(scene), xticklabels=xticklabels(scene))
```

Comment:
```suggestion
function xticks!(scene = current_scene(); xtickrange=xtickrange(scene), xticklabels=xticklabels(scene))
```
so on and so forth...

Reply:
Are keyword arguments alright here? It was the simplest solution I had for managing setting the need to set both the range and labels at the same time.

Reply:
Yeah, kwargs should be fine.  Note that there's a new tick system in MakieLayout which sort of changes the internals of the function (though not the method signature).

Reply:
I'm assuming that the need to change both will remain in the new system?

Reply:
Yeah, you still will be able to change both, but once you do, zooming won't work properly until you reset the ticks to the default.